modules/ob-example-configuration-for-defining-parameter-values.adoc

```diff
@@ -16,147 +16,131 @@ The following example shows a `ClusterBuildStrategy` CR that defines several par
 
 [source,yaml]
 ----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: buildkit-configuration
-  namespace: a-namespace
-data:
-  cache: disabled
+apiVersion: v1 
+kind: ConfigMap 
+metadata: 
+  name: buildah-configuration
+  namespace: <your_namespace>
+data: 
+  storage-driver: overlay
 ----
 
 You can use the created `ConfigMap` CR as a parameter value in your `Build` CR, as shown in the following example:
 
 [source,yaml]
 ----
-apiVersion: shipwright.io/v1beta1
-kind: Build
-metadata:
-  name: a-build
-  namespace: a-namespace
-spec:
-  paramValues:
-  - name: cache
-    configMapValue:
-      name: buildkit-configuration
-      key: cache
-  strategy:
-    name: buildkit
-    kind: ClusterBuildStrategy
-  source:
-  # ...
-  output:
+apiVersion: shipwright.io/v1beta1 
+kind: Build 
+metadata: 
+  name: <your_build> 
+  namespace: <your_namespace> 
+spec: 
+  paramValues: 
+  - name: storage-driver 
+    configMapValue: 
+      name: buildah-configuration 
+      key: storage-driver 
+  strategy: 
+    name: buildah 
+    kind: ClusterBuildStrategy 
+  output: 
   # ...
 ----
 
-.Example: Defining a `build-args` parameter in a `Build` CR
-You can define a `build-args` parameter as an array. For example, if you use the `buildkit` strategy, you can define a `build-args` parameter to set the argument values in the container file. Values must be defined in the `KEY=VALUE` format as shown in the following example:
+.Example: Assigning value to a parameter of the type `array` in a `Build` CR
+You can assign value to a parameter of the type `array`. If you use the `buildah` strategy, you can define a `registries-search` parameter to search images in specific registries. The following example shows how you can assign a value to the `registries-search` array parameter:
 
 [source,yaml]
 ----
-apiVersion: shipwright.io/v1beta1
-kind: Build
-metadata:
-  name: a-build
-  namespace: a-namespace
-spec:
-  paramValues:
-  - name: cache
-    configMapValue:
-      name: buildkit-configuration
-      key: cache
-  - name: build-args
+apiVersion: shipwright.io/v1beta1 
+kind: Build 
+metadata: 
+  name: <your_build> 
+  namespace: <your_namespace> 
+spec: 
+  paramValues: 
+  - name: storage-driver 
+    configMapValue: 
+      name: buildah-configuration 
+      key: storage-driver
+  - name: registries-search
     values:
-    - value: NODE_VERSION=16
-  strategy:
-    name: buildkit
-    kind: ClusterBuildStrategy
-  source:
-  # ...
-  output:
+    - value: registry.redhat.io 
+  strategy: 
+    name: buildah 
+    kind: ClusterBuildStrategy 
+  output: 
   # ...
 ----
 
-.Example: Referencing a config map or secret in a `Build` CR
-You can also reference config maps or secrets in the `build-args` array parameter as shown in the following example:
+.Example: Referencing a secret in a `Build` CR
+You can reference a secret in an array parameter as shown in the following example:
```

Comment:
```suggestion
You can reference a secret for an array parameter as shown in the following example:
```
I think `for` might suit here better. Let me know your thoughts, if otherwise.